projects/addon-editor/extensions/highlight/highlight.extension.ts

```diff
@@ -0,0 +1,40 @@
+import {getHTMLFromFragment} from '@tiptap/core';
+import {Highlight} from '@tiptap/extension-highlight';
+
+export const TuiHighlight = Highlight.extend({
+    addKeyboardShortcuts(): any {
+        return {
+            Tab: () => {
+                if (this.editor.isActive(`highlight`)) {
```

Comment:
Can you explain this code a little bit? Why `Tab` and `Enter` have some specific actions inside `mark`?

Reply:
fixed